site2/website/release-notes.md

```diff
@@ -1,6 +1,300 @@
 
 ## Apache Pulsar Release Notes
 
+### 2.10.0
+#### 2022-02-28
+
+### Update notice
+Remove -XX:-ResizePLAB JVM option which degrades performance on JDK11 [#12940](https://github.com/apache/pulsar/pull/12940)
+Enable TCP keepAlive flag on the sockets [#12982](https://github.com/apache/pulsar/pull/12982)
+Reduce the time it takes for namespace bundle unloading to time out [#12995](https://github.com/apache/pulsar/pull/12995)
+Align configurations defaults between default file and Java object (broker.conf, proxy.conf, websocket.conf) [#13272](https://github.com/apache/pulsar/pull/13272)
+[PIP 118] Do not restart brokers when ZooKeeper session expires as default [#13341](https://github.com/apache/pulsar/pull/13341)
+[PIP 119] Enabled consistent hashing by default on KeyShared subscriptions dispatcher [#13352](https://github.com/apache/pulsar/pull/13352)
+[PIP 120] Enabled client memory limit controller by default [#13344](https://github.com/apache/pulsar/pull/13344)
+Make Docker images non-root, by default, and OpenShift compliant [#13376](https://github.com/apache/pulsar/pull/13376)
+[PIP 122] Change loadBalancer default loadSheddingStrategy to ThresholdShedder [#13733](https://github.com/apache/pulsar/pull/13733)
+Fixed netcat returning early for probe [#14088](https://github.com/apache/pulsar/pull/14088)
```

Comment:
Some items are in the past tense while others are in the present tense. Please keep the same tense for all items.